### DISCUSSION TITLE
Covenant CoC

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#13] [Q&A] [answered]
The CoC is one of the thing imposed by the foundation on the projects that otherwise have governance free reign. That feels at odds with the foundation objectives.
Rational?

**Comment by ebullient** [comment, marked as answer]
To grow communities these days, you need a CoC:

<img width="815" alt="image" src="https://github.com/commonhaus/foundation-draft/assets/808713/1bbd2f6d-e4ad-4db6-b0ee-44f9323d5168">

BUT it is often hard to establish a decent enforcement/appeal process, especially if there are few maintainers. This is something I think the council can and should help with. There are linked resources on that page to help, for example: https://medium.com/mozilla-open-innovation/how-were-making-code-of-conduct-enforcement-real-and-scaling-it-3e382cf94415

I should revise this template a bit more, but the idea is that a project can use a template like this: https://github.com/commonhaus/foundation-draft/blob/main/templates/CODE_OF_CONDUCT.md, and use the foundation and the council to create a more robust reporting and appeal process.

There are questions on how to ensure CoC reports are handled fairly, and whether or not folks on the council need training... those are good questions. I guess I optimistically assume that most folks working in this area want to do the right thing (I've certainly observed that, so I feel like there is some basis in reality here), but I also recognize that we do need some teeth here.
